grails-app/controllers/org/pih/warehouse/api/LocalizationApiController.groovy

```diff
@@ -58,7 +59,58 @@ class LocalizationApiController {
         Locale locale = localizationService.getLocale(languageCode)
         String message = messageSource.getMessage(params.id, params.list("args").toArray(), locale)
         render([code: params.id, message: message, currentLocale: locale] as JSON)
-
     }
 
+    static JSON localizeHelpScoutLabels(GrailsApplication app, Locale locale) {
+        return [
+            "labels": [
+                /* fields in the Contact Us pane */
+                "messageButtonLabel"     : app.mainContext.getMessage("helpscout.beacon.contact.messageButtonLabel.label", null, locale),
```

Comment:
Is messageButtonLabel the name that helpscout is expecting in the config?

If so you could probably do this in a more systematic way.

message.properties
```
helpscout.messageButtonLabel.label = ...
helpscout.wereHereToHelp.label = ...
```

Code to build config that can be returned by the
```
def getHelpScoutConfig() { 
    Map configMap = [:]
    // LocalizationApiController could be refactored so we can move most of the code to messageService
    List messageProperties = messageService.getMessagePropertiesWithPrefix("helpscout")
    messageProperties.collect { it ->  
        // Property name is messageButtonLabel, wereHereToHelp, etc
        // You could also pass a regex with pattern matching instead of a prefix so that you can 
        // just pluck out the matching pattern out as the property name 
        String propertyName = extractPropertyName(it.key)   // remove helpscout and label
        configMap(propertyName, it.value)
    }    
    return configMap
```